.github/workflows/gito-code-review.yml

```
@@ -0,0 +1,57 @@
+    - uses: actions/checkout@v4
+      with: { fetch-depth: 0 }
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with: { python-version: "3.13" }
+
+    - name: Fetch Latest Gito Version
+      id: gito-version
+      run: pip index versions gito.bot 2>/dev/null | head -1 | sed -n 's/.* (\([^)]*\)).*/version=\1/p' >> $GITHUB_OUTPUT
+
+    - uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)